contrib/terraform/openstack/modules/compute/main.tf

```diff
@@ -167,9 +249,46 @@ resource "openstack_compute_instance_v2" "k8s_master_no_etcd" {
   }
 }
 
+resource "openstack_compute_instance_v2" "k8s_master_no_etcd_custom_volum_size" {
```

Comment:
Looks like a typo, should be k8s_master_no_etcd_custom_volume_size  (same with the 2 other instances of this variable).